mitodl/open-discussions
# Course detail view

Based on PR #1784

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1820 
Closes #1886

#### What's this PR do?
- Adds a right-side drawer (`static/js/containers/CourseDrawer.js`) containing a new `ExpandedCourseDetailComponent` with additional details on a single course selected from the list of search results.
 
#### How should this be manually tested?
Assuming courses have already been imported and indexed:
- Go to `/courses/search` and search for something like `Spatial database`
- Click on a search result
- A course drawer should open with additional details on the course
- Click the links to take the course, they should open the course page in a new tab.
- Click on the drawer's upper right X or somewhere outside the drawer, it should close.
- Click on another search result, the drawer should reopen with that course's details shown.


#### Screenshots

MITX:

![screen shot 2019-03-01 at 1 20 39 pm](https://user-images.githubusercontent.com/187676/53657719-eb8ebc00-3c24-11e9-9109-f972d1c6f925.png)

![screen shot 2019-03-01 at 1 23 51 pm](https://user-images.githubusercontent.com/187676/53657855-550eca80-3c25-11e9-894d-c1bf81f9fb3e.png)


OCW:

![screen shot 2019-03-01 at 1 25 02 pm](https://user-images.githubusercontent.com/187676/53657937-85566900-3c25-11e9-8842-f10f20560f1f.png)

![screen shot 2019-03-01 at 1 25 20 pm](https://user-images.githubusercontent.com/187676/53657956-89828680-3c25-11e9-8939-69b3cfbf0f75.png)
